lib/DraggableCore.es6

```diff
@@ -331,6 +345,10 @@ export default class DraggableCore extends React.Component {
     // We're on a touch device now, so change the event handlers
     dragEventFor = eventsFor.touch;
 
+    // Remove the event listener that stopped document scrolling
+    document.removeEventListener('touchmove', this.removeScroll, false);
```

Comment:
OK, it works without disturbing internal scrolling for us. I did use the draggableCancel and draggableHandle properties though. 

We will do some testing without those to see if it could be an issue for others. Thanks so much for your great product.